includes/wf_crm_webform_postprocess.inc

```diff
@@ -747,8 +747,8 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
    */
   private function saveContactLocation($contact, $cid, $c) {
     // Check which location_type_id is to be set as is_primary=1;
-    $is_primary_address_location_type = wf_crm_aval($contact, 'address:1:location_type_id');
-    $is_primary_email_location_type = wf_crm_aval($contact, 'email:1:location_type_id');
+    //$is_primary_address_location_type = wf_crm_aval($contact, 'address:1:location_type_id');
+    //$is_primary_email_location_type = wf_crm_aval($contact, 'email:1:location_type_id');
 
```

Comment:
Can you please remove these two lines.

Reply:
@KarinG done

Reply:
@KarinG please let us know if there is anything more we can do on this to close it off? many thanks!